README.md

```diff
@@ -98,8 +87,24 @@ Client instance ready for communication with the given
 The list of collected metrics is described in [METRICS.md](METRICS.md).
 
 ### Examples
+The example can be run on 3 different levels:
```

Comment:
Instead of "levels" here, can we refer to them as scenarios or configurations? I'm thinking: 

`This example can be run in 3 different scenarios:`